docs/process/teams.md

```diff
@@ -0,0 +1,13 @@
+These teams are not set in stone and will likely evolve as we continue to iterate on what makes the most sense for the success of the product.
+
+| Team | Purpose |
+|:--|:--|
+| `@desktop/analytics` | Analyzing the data we use for gaining insight into Desktop and its users |
+| `@desktop/maintainers` | The maintainers of GitHub Desktop |
+| `@desktop/comrades` | Avenue for community members to get more involved in the Desktop project |
+| `@desktop/docs` | Responsible for creating documentation Desktop users can consult when needed |
```

Comment:
Should we point out that we mean the docs here: https://help.github.com/desktop/? 
When I initially read this, I thought this might apply to repo READMEs or the marketing website.